package.json

```diff
@@ -77,8 +79,7 @@
     "tailwind-config-viewer": "^1.5.1",
     "tailwindcss": "npm:@tailwindcss/postcss7-compat",
     "tailwindcss-pseudo-elements": "^1.5.1",
-    "ts-node": "^8.6.2",
-    "webpack-bundle-analyzer": "^4.4.0"
```

Comment:
@nud3l 
May I ask why it was dropped?

Reply:
Just removed it for debugging purposes - readded now.